plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildInputs/ApiParameter.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Microsoft.DocAsCode.Common.EntityMergers;
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.YamlSerialization;
+
+    using Newtonsoft.Json;
+    using YamlDotNet.Serialization;
+
+    [Serializable]
+    public class ApiParameter
+    {
+        [YamlMember(Alias = "id")]
+        [JsonProperty("id")]
+        [MergeOption(MergeOption.MergeKey)]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// parameter's types
+        /// multiple types is allowed for a parameter in languages like JavaScript
+        /// </summary>
+        [YamlMember(Alias = "type")]
+        [JsonProperty("type")]
+        [UniqueIdentityReference]
+        public List<string> Type { get; set; }
+
+        [YamlMember(Alias = "description")]
+        [JsonProperty("description")]
+        [MarkdownContent]
+        public string Description { get; set; }
```

Comment:
it's parameter's description

---
In reply to: [112862172](https://github.com/dotnet/docfx/pull/1574#discussion_r112862172) [](ancestors = 112862172)